src/shogun/base/base_types.h

```diff
@@ -52,7 +53,8 @@ namespace shogun
 	                    std::is_same<CLikelihoodModel, T>::value ||
 	                    std::is_same<CMeanFunction, T>::value ||
 	                    std::is_same<CLossFunction, T>::value ||
-	                    std::is_same<CTokenizer, T>::value>
+	                    std::is_same<CTokenizer, T>::value ||
+						std::is_same<CEvaluationResult, T>::value>
```

Comment:
whitespace

Reply:
This will be fixed by the code formatter.

Reply:
but you need to apply it or?

Reply:
That will be the last thing. I do not want to pollute the diffs with whitespaces changes ;)